proxy_test.go

```diff
@@ -168,7 +168,7 @@ func TestReverseProxy_ServeHTTP1(t *testing.T) {
 			expStatusCode: http.StatusTooManyRequests,
 			f: func(p *reverseProxy) *http.Response {
 				p.clusters["cluster"].users["web"].maxConcurrentQueries = 1
-				go makeHeavyRequest(p, time.Millisecond*20)
+				go makeHeavyRequest(p, time.Millisecond*100)
 				time.Sleep(time.Millisecond * 10)
 				return makeRequest(p)
```

Comment:
Both of the modified tests asses rejection of the request when `maxConcurrentQueries` number is  exceeded. They were failing because of the slow environment. The query that was supposed to run for 20 milliseconds was too short for the CI. It was finishing before the 2nd query would arrive